app/javascript/dashboard/routes/dashboard/settings/reports/ReportContainer.vue

```diff
@@ -27,7 +31,16 @@ export default {
       }),
     },
   },
+  setup(props) {
+    const { calculateTrend, isAverageMetricType } = useReportMetrics(
+      props.accountSummaryKey
+    );
+    return { calculateTrend, isAverageMetricType };
+  },
   computed: {
+    ...mapGetters({
+      accountReport: 'getAccountReports',
```

Comment:
Why did we have to add this?

Reply:
I guess We are using this getter in this component.